recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set version = "6.1.3" %}
 
 package:
-  name: astropy
+  name: astropy-split
```

Comment:
Why the name change?

Reply:
just following instructions: https://conda-forge.org/docs/maintainer/knowledge_base/#common-pitfalls-with-outputs